Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/RdbmsProviderObjectMother.cs

```diff
@@ -36,7 +36,7 @@ public static class RdbmsProviderObjectMother
     public static RdbmsProvider CreateForIntegrationTest (
         IStorageSettings storageSettings,
         RdbmsProviderDefinition storageProviderDefinition,
-        Func<RdbmsProviderDefinition, IPersistenceExtension, IStorageProviderCommandFactory<IRdbmsProviderCommandExecutionContext>, RdbmsProvider> ctorCall = null)
+        Func<RdbmsProviderDefinition, string, IPersistenceExtension, IStorageProviderCommandFactory<IRdbmsProviderCommandExecutionContext>, RdbmsProvider> ctorCall = null)
```

Comment:
```suggestion
        Func<RdbmsProviderDefinition, IPersistenceExtension, IStorageProviderCommandFactory<IRdbmsProviderCommandExecutionContext>, RdbmsProvider> ctorCall = null)
```

We can simplify this. There's no usecase to have the ConnectioNString from passed via parameter.